RCNN_tf.py

```diff
@@ -311,10 +313,20 @@ def accuracy(
         result_
     )
 
+    tf.summary.scalar('accuracy', accuracy_)
+
+    summaries = tf.summary.merge_all()
     global_init_op_ = tf.global_variables_initializer()
     local_init_op_ = tf.local_variables_initializer()
 
     with tf.Session() as sess_:
+        #
+        # writer = tf.summary.FileWriter('logs/.')
+        # writer.add_graph(sess_.graph)
```

Comment:
we can leave it for now, but later we need to decide whether to use it or not (or we use a flag for indicating that)